vic/extensions/rout_irr/src/rout_alloc.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+#include <rout.h>
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_image.h>
+#include <netcdf.h>
+
+//define routing variables
+void rout_alloc(void){
+    extern rout_struct rout;
+    extern global_param_struct global_param;
+    extern domain_struct local_domain;
+    
+    rout.x_size=get_nc_dimension("/home/bram/VIC5.0Routing/VICTestSetup/input/VIC_Params/RVIC_input_NL.nc","lon");
```

Comment:
Dit is onder andere de reden waarom ik het in een aparte branch wilde hebben: normaal gesproken zijn hard gecodeerde paden uit den boze in de code.
Voor het gemak en het testen snap ik waarom je het nu wel doet hoor.